ext/opentelemetry-ext-system-metrics/src/opentelemetry/ext/system_metrics/__init__.py

```diff
@@ -0,0 +1,198 @@
+            name="system.net.bytes",
+            description="System network bytes",
+            unit="bytes",
+            value_type=int,
+            label_keys=self._labels.keys(),
+        )
+
+        self.meter.register_observer(
+            callback=self._get_runtime_memory,
+            name="runtime.python.mem",
+            description="Runtime memory",
+            unit="bytes",
+            value_type=int,
+            label_keys=self._labels.keys(),
+        )
+
+        self.meter.register_observer(
+            callback=self._get_runtime_gc_count,
+            name="runtime.python.gc.count",
+            description="Runtime: gc objects",
+            unit="objects",
+            value_type=int,
+            label_keys=self._labels.keys(),
+        )
+
+    def _get_system_memory(self, observer: metrics.Observer) -> None:
+        """Observer callback for memory available
+
+        Args:
+            observer: the observer to update
+        """
+        system_memory = psutil.virtual_memory()
+        _metrics = [
+            "total",
+            "available",
+            "used",
+            "free",
+        ]
+        for metric in _metrics:
+            self._system_memory_labels["type"] = metric
+            observer.observe(
+                getattr(system_memory, metric), self._system_memory_labels
+            )
+
+    def _get_system_cpu(self, observer: metrics.Observer) -> None:
+        """Observer callback for system cpu
+
+        Args:
+            observer: the observer to update
+        """
+        _metrics = [
+            "user",
+            "system",
+            "idle",
+        ]
+        cpu_times = psutil.cpu_times()
+        for metric in _metrics:
+            self._system_cpu_labels["type"] = metric
+            observer.observe(
+                getattr(cpu_times, metric), self._system_cpu_labels
+            )
+
+    def _get_network_bytes(self, observer: metrics.Observer) -> None:
+        """Observer callback for network bytes
+
+        Args:
+            observer: the observer to update
+        """
+        _metrics = [
+            "bytes_recv",
+            "bytes_sent",
+        ]
+        net_io = psutil.net_io_counters()
+        for metric in _metrics:
+            self._network_bytes_labels["type"] = metric
+            observer.observe(
+                getattr(net_io, metric), self._network_bytes_labels
+            )
+
+    def _get_runtime_memory(self, observer: metrics.Observer) -> None:
+        """Observer callback for runtime memory
+
+        Args:
+            observer: the observer to update
+        """
+        _metrics = [
+            "rss",
+            "vms",
+        ]
+        proc_memory = self._proc.memory_info()
+        for metric in _metrics:
+            self._runtime_memory_labels["type"] = metric
+            observer.observe(
+                getattr(proc_memory, metric), self._runtime_memory_labels
+            )
+
```

Comment:
I think adding process cpu and process memory would be very useful as well. (https://github.com/microsoft/opentelemetry-azure-monitor-python/blob/master/azure_monitor/src/azure_monitor/sdk/auto_collection/performance_metrics.py#L76)

Reply:
added runtime cpu as well